nextflow.config

```diff
@@ -9,14 +9,70 @@
 // Global default params, used in configs
 params {
 
-    // TODO nf-core: Specify your pipeline's command line flags
     // Input options
     input                      = null
 
     // References
     genome                     = null
     igenomes_base              = 's3://ngi-igenomes/igenomes'
     igenomes_ignore            = false
+    ucscname                   = null
+    mappability                = null
+    blacklist                  = null
+
+    //trim reads
+    skip_cutadapt             = false
+
+    // Enzyme
+    enzyme                     = 'CviQI'
+    restriction_sites          = '^TAC'
+    restriction_sites_cut_off  = 0.5
+
+    // ATAC peak calling
+    shiftsize                  = -75
+    smooth_window              = 150
+    qval_thresh                = 0.01
+
+    // R1 peak calling
+    high_resolution_R1         = false
+    r1_pval_thresh             = 0.1
+
+    // HiPeak calling
+    remove_dup                 = false
+    cluster_type               = "SOCK"
```

Comment:
Where is this parameter used? I couldn't find it easily

Reply:
The cluster_type will be used by the BiocParallel package (indeed it will pass to snow package) to handle multiple thread computation.